src/app.ts

```diff
@@ -828,6 +853,38 @@ function registerSessionAPIRoutes(
   );
 }
 
+function registerGeoAPIRoutes(
+  app: Express,
+  basePath: string,
+  geoService: GeoService,
+  hereApiKey: NonEmptyString,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  bearerSessionTokenAuth: any
+): void {
+  const geoController: GeoController = new GeoController(
+    geoService,
+    hereApiKey
+  );
+
+  app.get(
+    `${basePath}/geo/autocomplete`,
```

Comment:
Isn't `geo` already included in `basePath`

Reply:
Nope, basePath includes only `/api/v1`. It's similar to CGN schema